module/apmawssdkgo/session.go

```diff
@@ -32,10 +38,50 @@ func init() {
 	)
 }
 
+var (
+	inflight = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "apmawssdkgo",
+		Name:      "in_flight_requests",
+		Help:      "A gauge of in-flight AWS RPC calls, partitioned by service and rpc.",
+	},
+		[]string{"service", "rpc"},
+	)
+	receivedBytes = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "apmawssdkgo",
+			Name:      "http_received_bytes_total",
+			Help:      "A counter for the total requests bytes sent to AWS, partitioned by service, rpc, and code.",
+		},
+		[]string{"service", "rpc", "code"},
+	)
+	sentBytes = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "apmawssdkgo",
+			Name:      "http_sent_bytes_total",
+			Help:      "A counter for the total response bytes read from AWS, partitioned by service, rpc, and code.",
+		},
+		[]string{"service", "rpc", "code"},
+	)
+)
+
 // WrapSession wraps the provided AWS session with handlers that hook into the
 // AWS SDK's request lifecycle. Supported services are listed in serviceTypeMap
 // variable below.
-func WrapSession(s *session.Session) *session.Session {
+func WrapSession(s *session.Session, opts ...Option) *session.Session {
```

Comment:
I'm pretty strict on backwards compatibility: I don't think we'll be able to make this signature change, or the "go1.13" build tag change, once we've released the apmawssdkgo module. (Until we bump the major version that is.)

Reply:
Yeah makes sense. Let's keep it simple and remove this.